### DISCUSSION TITLE
How can I see my plan ?

### DISCUSSION THREAD

**Discussion by mehdi-wsc** [flux-iac/tofu-controller#458] [Q&A]
Hello Folks,
I would like to visualize my plan before apply, but unfortunately I don't find the generated plan.
Can you explain how to export the plan please ?
Best regards

**Comment by chanwit** [comment]
Thank you for the question, @mehdi-wsc.
We would have added this to the documents. I'll do it for the next release.

First, please install the client: `brew install weaveworks/tap/tfctl`

Second, please add:

```yaml
spec:
   storeReadablePlan: human
```
to your object.

Then when it says: "Plan generated ...", please run:

```
tfctl show plan <your object name>
```
to view the plan.
